src/components/Navbar.js

```diff
@@ -10,7 +10,11 @@ function Navbar() {
         <div className={NavbarCSS.logo}>Bookstore CMS</div>
         <Link className={NavbarCSS.navLink} to="/">Books</Link>
         <Link className={NavbarCSS.navLink} to="/categories">Categories</Link>
-        <span className="material-symbols-rounded user-icon" />
+        <div className="avatar">
+          <div className="user-icon">
+            <span />
+          </div>
+        </div>
```

Comment:
- Great adding the icon, however, you've not aligned the icon correctly which should be at the extreme end of the right, and also, you've not used the correct color on the icon. I understand you've put in more effort. please kindly update this issue pointed out. I'd approve your project. Kindly keep in mind that being attentive to detail is a skill that would make you standout in the job market, let's improve on this skill :pray:

Reply:
thanks for your valuable comment sure i will be doing it